README.md

```diff
@@ -47,38 +48,56 @@ some window managers (i3, sway…):
   - [ ] Abanu(Desktop Environment for Windows and Linux)
   - [ ] Pantheon 
   - [ ] TOS Desktop Environment
+  - [ ] Unity
   - [ ] Trinity Desktop Environment
   - [ ] Laniakea Desktop Environment
   - [ ] AtomOS Desktop Environment
   - [x] Budgie
+  - [ ] Common Desktop Environment
   - [x] Cinnamon
+  - [ ] EDE
+  - [ ] GNUstep
+  - [ ] Manokwari
+  - [ ] Razor-qt
+  - [ ] Rox Desktop
   - [x] Deepin
   - [ ] **WIP** Enlightenment (need to proceed through setup wizard)
+  - [ ] Windowmaker
+  - [ ] Ratpoison
```

Comment:
From what I can remember, Ratpoison is more of a window manager than a full desktop environment